core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -32,6 +32,8 @@ import { AlertsService } from 'services/alerts.service';
 import { QuestionsListService } from 'services/questions-list.service';
 import { LoaderService } from 'services/loader.service';
 
+type assignedSkillTopicDataType = { [topicName: string]: string } | null;
```

Comment:
```suggestion
type AssignedSkillTopicData = { [topicName: string]: string };
```
lets do the `nuzll` part down below explicitely